src/Test.UnitTests.BinSkim.Driver/CompilerDataLoggerTests.cs

```diff
@@ -70,72 +114,46 @@ public void CompilerDataLogger_Constructor_ShouldThrowException_WhenForceIsDisab
             };
 
             Assert.Throws<InvalidOperationException>(() => new CompilerDataLogger(SarifPath, Sarif.SarifVersion.Current, context, fileSystem.Object));
+            context.Dispose();
         }
 
         [Fact]
-        public void CompilerDataLogger_Constructor_ShouldNotThrowException_WhenForceAndCsvAreEnabled()
+        public void CompilerDataLogger_Constructor_ShouldNotLogEvents_WhenForceAndCsvAreEnabled()
```

Comment:
>ShouldNotLogEvents_WhenForceAndCsvAreEnabled [](http://example.com/codeflow?start=51&length=44)

Two tests here to verify telemetry is logged when telemetry enabled in this scenario, and doesn't when telemetry is disabled.

Reply:
I went back through the history of this test. Reverted part of name change and merged with Michael's suggestion to properly capture the purpose of this test.